packages/material-ui-lab/src/ToggleButtonGroup/ToggleButtonGroup.js

```diff
@@ -65,6 +65,7 @@ const ToggleButtonGroup = React.forwardRef(function ToggleButton(props, ref) {
         return React.cloneElement(child, {
           selected,
           onChange: exclusive ? handleExclusiveChange : handleChange,
+          size: props.size ? props.size : 'medium',
```

Comment:
```suggestion
          size,
```

(The default is already defined.)